### DISCUSSION TITLE
How to use?

### DISCUSSION THREAD

**Discussion by TackyBoxer** [YimMenu/YimMenu#1072] [Q&A]
I've just downloaded all the necessary YimMenu files from here and followed all the instructions regarding building the project but I don't understand how I actually use it for GTA. Previously I tried just injecting the DLL file but that crashed my game and I am not sure how to use this.

**Comment by Seanghost117** [comment]
I hate to be that guy but if you are attempting to build and make use of the project then you have to first understand what you are working with and how to use it. My first instinct is to tell you to simply debug, you downloaded the project, straight built it, and injected it so your first job is to debug any errors before asking for help because we also can't just make assumptions for you. Firstly check your logs, and see where the crash happens, and what file or line the exception is related to. Also, consider whether have you tried to change anything. This also goes back to understanding what you are working with, if you change things without understanding them the result is not what you may like. Also, consider any requirements, there are translation files that are required. When are you crashing? Does it successfully inject and crash on opening or before then? 

Also to clarify are you saying that you downloaded the nightly build and it crashes and you are having trouble with building the project or the binary that you built from the project is crashing? You are going to have to clarify a lot more points.

**Reply by TackyBoxer** [reply]
I appreciate your reply. For quite a while now I have been trying to inject just the dll file without any git files. Each time I try, there is no error message, all seems to go well and the injection is successful. However, after around 5 seconds, the game crashes and I do not get an error code or anything. I thought using the git project would help, I followed all the steps and built everything but I am not sure how to run the project. The first few times, the dll file worked fine when injecting, I am not sure what happened. Initially I did have errors which were fixed after running the injector as admin. Unfortunately that was not the fix this time.

**Reply by Seanghost117** [reply]
Firstly make sure you are using the nightly build. Also have you checked your logs yet? They can be found by pressing windows key + r and typing appdata, go to roaming and find Big Base v2 and find the cout.log in the main directory, you can also find logs for previous sessions in backups.

**Reply by TackyBoxer** [reply]
I appreciate that but I still am not sure how I would run YimMenu through the GitHub build, I have built everything but I am unsure of how to run it.

**Reply by Seanghost117** [reply]
The same way you inject the dll, its found in the root directory wherever you are keeping it -> out -> build and you will find your compiled dll

**Comment by TackyBoxer** [comment]
I’ll have to check that out, thank you, I didn’t notice any files like that. Would they appear in a specific folder?

**Reply by Seanghost117** [reply]
YimMenu > out > build > x64-Release > YimMenu.dll

**Reply by TackyBoxer** [reply]
You’re a legend, thank you so much!

**Reply by Seanghost117** [reply]
No Problem
